creme/creme_config/forms/fields.py

```diff
@@ -167,3 +168,66 @@ def __init__(self, *, custom_field=None, user=None, **kwargs):
         super().__init__(coerce=int, **kwargs)
         self.custom_field = custom_field
         self.user = user
+
+
+class BricksConfigField(fields.JSONField):
+    widget = BricksConfigWidget
+
+    zones = ('top', 'left', 'right', 'bottom')
+    default_error_messages = {
+        'invalid_format': _("The value doesn't match the expected format."),
+        'invalid_choice': (
+            fields.ChoiceField.default_error_messages['invalid_choice']),
+    }
+
+    def __init__(self, *, choices=(), **kwargs):
+        kwargs.setdefault('initial', {zone: [] for zone in self.zones})
```

Comment:
As the default value is ""heavy"", I'd prefer a `if 'initial' not in kwargs: ...`